deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -159,6 +159,42 @@ spec:
             online:
               description: Should the server be online?
               type: boolean
+            rootDeviceHints:
+              description: What user defined root device hints is used for this host?
+                User can optionally define the content of the root device hints
+              properties:
+                devicename:
+                  description: A device name like "/dev/vda"
+                  type: string
+                hctl:
+                  description: A SCSI bus address like 0:0:0:0
+                  type: string
+                model:
+                  description: Device identifier
+                  type: string
+                rotational:
+                  description: Device rotational type
+                  type: boolean
+                serial:
+                  description: Disk serial number
+                  type: string
+                sizeBytes:
```

Comment:
why not just size, and document that you need to specify on gigabytes? Also, it's confusing as you say "bytes"